Purchases/Public/RCPurchasesErrors.h

```diff
@@ -44,6 +44,7 @@ typedef NS_ERROR_ENUM(RCPurchasesErrorDomain, RCPurchasesErrorCode) {
     RCNetworkError,
     RCInvalidCredentialsError,
     RCUnexpectedBackendResponseError,
+    RCReceiptInUseByOtherSubscriberError __attribute((deprecated("Use RCReceiptAlreadyInUseError."))),
```

Comment:
I might be wrong, but I think we have to add it to Android so we have parity in the hybrid SDKs. What I mean is if this is error code 12, it also needs to be the error 12 in Android. If we don't have this error in Android, RCInvalidAppUserIdError will be 12 in Android. This is bad and we should fix it by setting fixed codes instead of relying on the size and order of the enum.

Reply:
let's go ahead and add the numbers explicitly. I'll add them to this PR, we can do the same on android and make sure to match them